src/components/Sketches/index.js

```diff
@@ -51,7 +49,10 @@ class Sketches extends React.Component {
   };
 
   setProgram = (name) => {
-    this.props.setMostRecentProgram(name);
+    const { uid } = this.props;
+
+    // eslint-disable-next-line react/destructuring-assignment
```

Comment:
I believe this can be removed

Reply:
Hi Timothy, thank you for the review! I fixed the other issues just now, but I'm not very sure about what this one is referring to. Do you mean to remove the destructuring of `uid` and use `this.props.setMostRecentProgram(name, this,props.uid)` directly in the next line?